### PR TITLE
Change S3 bucket urls to new method as per #56124

### PR DESCRIPTION
**Pull request by ShachafGoldstein**
##### SUMMARY
Change the url for S3 buckets as mentioned in https://github.com/ansible/ansible/issues/56124

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME
test/integration/targets/setup_win_psget/tasks/main.yml
test/integration/targets/win_command/tasks/main.yml
test/integration/targets/win_get_url/tasks/main.yml
test/integration/targets/win_get_url/tasks/tests_url.yml
test/integration/targets/win_hotfix/defaults/main.yml
test/integration/targets/win_module_utils/tasks/main.yml
test/integration/targets/win_package/defaults/main.yml
test/integration/targets/win_pester/tasks/main.yml
test/integration/targets/win_psexec/tasks/main.yml
test/integration/targets/win_psmodule/tasks/setup.yml
test/integration/targets/win_service/defaults/main.yml
test/runner/setup/windows-httptester.ps1

### CODE REVIEW

File: test/integration/targets/setup_win_psget/tasks/main.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@
   block:
   - name: install PackageManagement
     win_package:
-      path: https://s3.amazonaws.com/ansible-ci-files/test/integration/targets/setup_win_psget/PackageManagement_x64.msi
+      path: https://ansible-ci-files.s3.amazonaws.com//test/integration/targets/setup_win_psget/PackageManagement_x64.msi
```
Comment (mattclay) [inline test/integration/targets/setup_win_psget/tasks/main.yml]: Replace `//` with `/` here and for all following changes:

```suggestion
      path: https://ansible-ci-files.s3.amazonaws.com/test/integration/targets/setup_win_psget/PackageManagement_x64.msi
```

**Review by jborean93 — COMMENTED** [review]
I'm pretty sure it will still work but you've left the `/` after the `s3.amazonaws.com` which has resulted in a duplicated char. Would be good to have that removed.

Edit: mattclay beat me to it

File: test/integration/targets/setup_win_psget/tasks/main.yml (hunk 1 above)
Comment (ShachafGoldstein) [reply test/integration/targets/setup_win_psget/tasks/main.yml]: fixed thanks

File: changelogs/fragments/56418-S3-buckets-path-change [hunk 2]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- Implement new supported path for bucket access (https://github.com/ansible/ansible/issues/56124)
```
Comment (mattclay) [inline changelogs/fragments/56418-S3-buckets-path-change]: Changelog fragments aren't needed for test changes.

**Comment by ShachafGoldstein** [comment]
the duplicate char actually broke the CI

File: changelogs/fragments/56418-S3-buckets-path-change (hunk 2 above)
Comment (ShachafGoldstein) [reply changelogs/fragments/56418-S3-buckets-path-change]: K

File: test/integration/targets/setup_win_psget/tasks/main.yml [hunk 3]
```diff
@@ -10,7 +10,7 @@
   block:
   - name: install PackageManagement
     win_package:
-      path: https://s3.amazonaws.com/ansible-ci-files/test/integration/targets/setup_win_psget/PackageManagement_x64.msi
+      path: https://ansible-ci-files.s3.amazonaws.comtest/integration/targets/setup_win_psget/PackageManagement_x64.msi
```
Comment (mattclay) [inline test/integration/targets/setup_win_psget/tasks/main.yml]: Now the `/` after the domain name is missing:

```suggestion
      path: https://ansible-ci-files.s3.amazonaws.com/test/integration/targets/setup_win_psget/PackageManagement_x64.msi
```

**Review by mattclay — APPROVED** [review]

**Comment by jborean93** [comment]
Test failures are unrelated.

**Pull request merged**